examples/3d/occlusion_culling.rs

```diff
@@ -185,6 +190,10 @@ fn main() {
                 .set(RenderPlugin {
                     allow_copies_from_indirect_parameters: true,
                     ..default()
+                })
+                .set(PbrPlugin {
+                    allow_copies_from_indirect_parameters: true,
```

Comment:
Could this be reused from the RenderPlugin? Rather than having to set it in two places.

Reply:
Could you elaborate as to how that would work? The problem is that the `PbrPlugin` can't reach into the `RenderPlugin` to check its value.

Reply:
I think this is fine for now but it's a rather particular option and I might suggest subsuming it some kind of broader "debug renderer" setting if we ever have a second instance of this kind of thing.

Reply:
Yeah, good point. Or maybe a "debug flags"?

Reply:
I went ahead and switched this to a `RenderDebugFlags` so that we can have more of them.